contracts/external/pooltogether/FixedPoint.sol

```diff
@@ -0,0 +1,75 @@
+/**
+Copyright 2020 PoolTogether Inc.
+
+This file is part of PoolTogether.
+
+PoolTogether is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation under version 3 of the License.
+
+PoolTogether is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with PoolTogether.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.6.12;
```

Comment:
I think this should be given a range too, for 3rd-party projects that import PoolTogether repos